src/main/scala/org/kframework/minikore/implementation/outer.scala

```diff
@@ -0,0 +1,45 @@
+package org.kframework.minikore.implementation
+
+
+/**
+  * Created by lpena on 3/23/17.
+  */
+object outer {
```

Comment:
outer should extend something an OuterBuilders trait, like DefaultBuilders extends the Builders trait for pattern